shuffle/cli/src/shared.rs

```diff
@@ -35,3 +39,9 @@ pub fn send(client: &BlockingClient, tx: diem_types::transaction::SignedTransact
     );
     Ok(())
 }
+
+// returns ~/.shuffle
+pub fn get_shuffle_dir() -> PathBuf {
+    let dir = BaseDirs::new().unwrap().home_dir().join(".shuffle");
+    dir
+}
```

Comment:
Let's add a unit test to get coverage on this.

Reply:
on it!